cli/lib/tasks/process-js-task.js

```diff
@@ -110,7 +113,11 @@ class ProcessJsTask extends IncrementalFileTask {
 			return this.doFullTaskRun();
 		}
 
-		Object.keys(this.data.jsFiles).forEach(relPath => this.builder.unmarkBuildDirFile(this.data.jsFiles[relPath].dest));
+		this.jsFiles = this.data.jsFiles;
```

Comment:
OK, so looking at the code, it appears we already compare hashes inside the hooks that processJsFile does, so we effectively skip the work on them anyways?

Reply:
Yes, it will still skip unchanged files. It just goes through the processing flow to make sure the hooks are called and that the builder gets populated with required info about encrypted files.